### DISCUSSION TITLE
IDL protobuf stub模式下，客户端启动报错

### DISCUSSION THREAD

**Discussion by freemanxzx** [apache/dubbo#13839] [Question] [answered]
采用IDL protobuf，使用stub模式的时候，服务端能启动起来，客户端启动会报以下找不到DubboGreeterTriple这个类的错误，但protobuf自动生成了这个类的。

Exception in thread "main" java.lang.NoClassDefFoundError: Could not initialize class com.tongyuan.mworkscloud.licence.DubboGreeterTriple

另外，编译DubboGreeterTriple的时候，会提示“使用了未经检或不安全的操作。”， “有关详细信息, 请使用 -Xlint:unchecked 重新编译。”

用的DUBBO3框架

请教各位是什么原因导致的呢？

**Comment by freemanxzx** [comment, marked as answer]
已解决，版本的问题，建议官方出个pom，维护版本统一性

**Reply by AlbumenJ** [reply]
Would you like to enhance it?
